kyo-data/shared/src/main/scala/kyo/Text.scala

```diff
@@ -230,6 +232,17 @@ object Text:
 
         def compact: Text = self.toString()
 
+        def is(other: Text): Boolean =
```

Comment:
New method to check for equality. I've just noticed that `Text` is missing scaladocs, I'll follow up separatelly.